samples/bluetooth/broadcast_audio_source/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Bluetooth: Broadcast Audio Source"
+
+config BT_24KHZ_FREQUENCY
+	bool "Enable 24kHz as frequency configuration used"
+	default n
+	help
+	  Enable using 24kHz frequency configuration for bluetooth communication.
```

Comment:
Suggest to use `choice` so that we can expand it with 32, 48, etc. 

Since frequency is not the only thing we may need/want to change, maybe we should implement it as choice of the Set Names from the BAP spec? e.g. 16_2_1, 24_1_1, 32_2_2, etc.